db/config.cc

```diff
@@ -1066,8 +1066,6 @@ db::config::config(std::shared_ptr<db::extensions> exts)
         "to make sure that trying to create an invalid view causes an error.")
     , reversed_reads_auto_bypass_cache(this, "reversed_reads_auto_bypass_cache", liveness::LiveUpdate, value_status::Used, false,
             "Bypass in-memory data cache (the row cache) when performing reversed queries.")
-    , enable_optimized_reversed_reads(this, "enable_optimized_reversed_reads", liveness::LiveUpdate, value_status::Used, true,
-            "Use a new optimized algorithm for performing reversed reads.")
```

Comment:
@mykaul can we verify that this option is indeed unused out in the field?

Reply:
I'm not sure we have the facility (right now). We could if we make an effort - but does it matter - what's the impact of removing it?

Reply:
It is enabled by default. When set to false, however, the reversed queries are performed as forward ones and wrapped into a reversing reader. According to my conversation with @denesb, it was the very first implementation of the reversed queries. It was kept as a back door option in case the reverse queries implementation does not work properly.

Reply:
If we haven't heard of anyone complaining and thus using the fallback, I think it's safe to drop it. (can we include a deprecation warning in 6.0 release notes - @tzach ?)

Reply:
> It is enabled by default. When set to false, however, the reversed queries are performed as forward ones and wrapped into a reversing reader. According to my conversation with @denesb, it was the very first implementation of the reversed queries. It was kept as a back door option in case the reverse queries implementation does not work properly.

So if we find issues in the field with native reverse queries as per this PR,
will there be a way to fallback to the current implementation (before this PR)?

Reply:
However, given there is a cluster feature for it, it's easy to add configuration too.

Reply:
And for the original question - I'm sure no one touched this configuration.